### ISSUE TITLE
[Feature] Prevent Uploading Old Files

### ISSUE THREAD

**Issue by mrtsglk** [pqina/filepond#902]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

I am listing the files I have uploaded. When I add a new file, the same files are uploaded again. What can I do to prevent it from reloading files that have already been uploaded?

### Describe the solution you'd like

Files uploaded as data-upload="false" can be prevented from being uploaded again.

### Describe alternatives you've considered

Files uploaded as data-upload="false" can be prevented from being uploaded again.

**Comment by NormandoHall** [comment]
Yes please. When I use "load" endpoint, the retrieve file is upload to the server again.

**Comment by rikschennink** [comment]
You have to mark already uploaded files as "local", meaning the file is already on the server.
https://pqina.nl/filepond/docs/api/instance/properties/#files

**Comment by NormandoHall** [comment]
@rikschennink yes, I marked as "local". Also I tried with restore endpoint, marking as "limbo", and the file is upload again. The only way to stop upload again is setting "instantUpload" to false.
